tools/sentry-release

```diff
@@ -0,0 +1,151 @@
+_arg_version_code=
+
+
+print_help()
+{
+	printf '%s\n' "This script creates a release in sentry. This file uses the configuration from ./.sentryclirc"
+	printf 'Usage: %s [-o|--version-code <arg>] [-h|--help] <package-name> <version-name> <sourcemap-directory>\n' "$0"
+	printf '\t%s\n' "<package-name>: package name under which to release"
+	printf '\t%s\n' "<version-name>: name of the version to release"
+	printf '\t%s\n' "<sourcemap-directory>: relative path to the directory where the sourcemap is"
+	printf '\t%s\n' "-o, --version-code: code of the version to release, this is only needed for react-native (no default)"
+	printf '\t%s\n' "-h, --help: Prints help"
+}
+
+
+parse_commandline()
+{
+	_positionals_count=0
+	while test $# -gt 0
+	do
+		_key="$1"
+		case "$_key" in
+			-o|--version-code)
+				test $# -lt 2 && die "Missing value for the optional argument '$_key'." 1
+				_arg_version_code="$2"
+				shift
+				;;
+			--version-code=*)
+				_arg_version_code="${_key##--version-code=}"
+				;;
+			-o*)
+				_arg_version_code="${_key##-o}"
+				;;
+			-h|--help)
+				print_help
+				exit 0
+				;;
+			-h*)
+				print_help
+				exit 0
+				;;
+			*)
+				_last_positional="$1"
+				_positionals+=("$_last_positional")
+				_positionals_count=$((_positionals_count + 1))
+				;;
+		esac
+		shift
+	done
+}
+
+
+handle_passed_args_count()
+{
+	local _required_args_string="'package-name', 'version-name' and 'sourcemap-directory'"
+	test "${_positionals_count}" -ge 3 || _PRINT_HELP=yes die "FATAL ERROR: Not enough positional arguments - we require exactly 3 (namely: $_required_args_string), but got only ${_positionals_count}." 1
+	test "${_positionals_count}" -le 3 || _PRINT_HELP=yes die "FATAL ERROR: There were spurious positional arguments --- we expect exactly 3 (namely: $_required_args_string), but got ${_positionals_count} (the last one was: '${_last_positional}')." 1
+}
+
+
+assign_positional_args()
+{
+	local _positional_name _shift_for=$1
+	_positional_names="_arg_package_name _arg_version_name _arg_sourcemap_directory "
+
+	shift "$_shift_for"
+	for _positional_name in ${_positional_names}
+	do
+		test $# -gt 0 || break
+		eval "$_positional_name=\${1}" || die "Error during argument parsing, possibly an Argbash bug." 1
+		shift
+	done
+}
+
+parse_commandline "$@"
+handle_passed_args_count
+assign_positional_args 1 "${_positionals[@]}"
+
+# OTHER STUFF GENERATED BY Argbash
+
+### END OF CODE GENERATED BY Argbash (sortof) ### ])
+# [ <-- needed because of Argbash
+
+set -e
+shopt -s nullglob
+
+if ! command -v sentry-cli &> /dev/null
+then
+  curl -sL https://sentry.io/get-cli/ | bash
+fi
+
+package_name=$_arg_package_name
+version_name=$_arg_version_name
+version_code=$_arg_version_code
+sourcemap_directory=$_arg_sourcemap_directory
+
+
+if [ -z "$SENTRY_AUTH_TOKEN" ]
+then
+  echo "SENTRY_AUTH_TOKEN is not set"
+  exit 1
+fi
+
+version="${package_name}@${version_name}"
+
+
+if [ ! -z "$version_code" ]
```

Comment:
I think there should be a version code all the time as well.

Reply:
I excluded the version_code from the version in the webapp as it has no valuable information

In our projects version_name = version_code